gems/railties/6.0/railties-generated.rbs

```diff
@@ -1434,16 +1434,8 @@ module Rails
     class MiddlewareStackProxy
       def initialize: (?untyped operations, ?untyped delete_operations) -> untyped
 
-      def insert_before: (*untyped args) { () -> untyped } -> untyped
-
-      alias insert insert_before
-
-      def insert_after: (*untyped args) { () -> untyped } -> untyped
-
       def swap: (*untyped args) { () -> untyped } -> untyped
 
-      def use: (*untyped args) { () -> untyped } -> untyped
-
       def delete: (*untyped args) { () -> untyped } -> untyped
 
       def unshift: (*untyped args) { () -> untyped } -> untyped
```

Comment:
Could you change `#swap`, `#delete`, and `#unshift` too?


Note:

The args and block are forwarded to the rack middleware class when building its instance. Therefore it's better to make them optional.

refs:

* https://github.com/rails/rails/blob/v6.0.6.1/railties/lib/rails/configuration.rb#L37
* https://github.com/rails/rails/blob/main/actionpack/lib/action_dispatch/middleware/stack.rb#L43-L45

Reply:
Thanks! I'll move those and make their arguments/block optional as well.